### DISCUSSION TITLE
I am making an application with nextjs, postgresql and prisma

### DISCUSSION THREAD

**Discussion by VictorDevCode21** [prisma/prisma#20351] [Q&A] [answered]
I need some help, my website is basically 2 admin panels for making changes in my database through APis and I need to know if I am working with prisma for free is enough to make this website works. I don't think that the website is going to have many requests, mostly is going to be 20 people and the only functionality is to update the available inventory. I don't know if I need to pay another version of prisma to do that or if it would works with the free version. Also, if I want to do websites like e-commerce with many traffic, is the paid version enough? what happens if I reach the limit of prisma? my website isn't going to work or do any requests?

**Comment by ludralph** [comment, marked as answer]
Hi @VictorDevCode21 👋 

[Prisma ORM](https://www.prisma.io/), which is the core component that allows you to interact with your database and perform Object-Relational Mapping, is available for free and can be used in your projects without any cost. It is sufficient for projects with low to moderate traffic, such as your website with around 20 people requesting to update inventory.

However, if you plan to scale your website to handle high traffic or build complex e-commerce applications with many concurrent users, you might consider using the [Prisma Data Platform](https://www.prisma.io/data-platform).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by VictorDevCode21** [reply]
Thank u so much, this helped me a lot!
